R/crs.R

```diff
@@ -147,7 +147,7 @@ make_crs = function(x) {
 #' @examples
 #' sfc = st_sfc(st_point(c(0,0)), st_point(c(1,1)))
 #' library(dplyr)
-#' x <- sfc %>% st_set_crs(4326) %>% st_transform(3857)
+#' x = sfc %>% st_set_crs(4326) %>% st_transform(3857)
```

Comment:
The code contains a mix of `=` and `<-`, any reason to favor `=`? @edzer, do you have a style you would like to impose? I think it would be good to adhere to a unique style; I prefer `<-`, but would commit to any.